pytest_selenium/pytest_selenium.py

```diff
@@ -93,13 +97,27 @@ def capabilities(request, driver_class, chrome_options, firefox_options,
         if all([key, options]):
             capabilities[key] = _merge(
                 capabilities.get(key, {}), options.get(key, {}))
-    capabilities_marker = request.node.get_marker('capabilities')
-    if capabilities_marker is not None:
+    capabilities_marker = _get_marker(request.node, 'capabilities')
+    LOGGER.debug('Capabilities from marker: {}'.format(capabilities_marker))
+    if capabilities_marker:
         # add capabilities from the marker
-        capabilities.update(capabilities_marker.kwargs)
+        capabilities.update(capabilities_marker)
     return capabilities
 
 
+def _get_marker(node, name):
```

Comment:
This isn't returning the markers, but a dictionary of the marker's keyword arguments. I appreciate your desire to make this a generic function, but I think in this case we could just name it `get_capabilities_from_markers`. We could also simplify to `capabilities.update(get_capabilities_from_markers(request.node))` as the return value will always be a dictionary so we don't need the `if x is not None` block.